libs/platform/user/kernel_um.cpp

```diff
@@ -66,39 +195,29 @@ unsigned long __cdecl DbgPrintEx(
 void
 ExInitializeRundownProtection(_Out_ EX_RUNDOWN_REF* rundown_ref)
```

Comment:
The analyze pass failed because rundown_ref is marked as an Out param in line 196 but 198 only uses it as an In parameter.